README.md

```diff
@@ -1,2 +1,47 @@
-## NCEP Implementation of FourCastNet model using GDAS data as ICs
+# NCEP Implementation of FourCastNet model using GDAS data as ICs
+We use ai_models_fourcastnetv2 plugin developed by ECMWF to run the model, which is FoureCastNet v2-small (https://arxiv.org/abs/2306.03838).
```

Comment:
@linlin did you fork the fourcastNet model code yet? Can we have a sentence here on FourCastNet model itself?

Reply:
@LinlinCui-NOAA @junwang-noaa it looks to me Linlin used ECMWF model for running fourcastnet, I think it would be better to get the Fourcastnet from its own repo and configure our script with the main code (like we did for GraphCast).

Reply:
@junwang-noaa @SadeghTabas-NOAA Scripts provided are to run FourCastNetv2, which cannot be found on NVIDIA's github. So I am using ECMWF's fourcastnetv2 plugin. FourCastNetv2 has 13 levels, while FourCastNet only has 3 levels.

Reply:
@LinlinCui-NOAA let me see if I can find it for you!

Reply:
@junwang-noaa I added the link to ECMWF's FourCastNetv2 model. This package is used to run the model. And I generated 10-day forecast data. Could you take a look to see if all variables look reasonable? You can find sample data on Hera:

```
/scratch1/NCEPDEV/nems/Linlin.Cui/Tests/FourCastNetv2/outputs_2024022100/
```
`fcngfs.t00z.pgrb2.0p25.f*`